src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/controller/OauthController.java

```diff
@@ -100,15 +100,15 @@ public void authenticate(HttpServletRequest request, HttpServletResponse respons
 		try {
 			token = oauthServices.getToken(authCode);
 		} catch (Exception e) {
-			logger.error("Error while calling Oauth2 /token endpoint. " + e.getMessage(), e);
+			logger.error("Error while calling Oauth2 /token endpoint. ", e);
 			return new ResponseEntity<String>(HttpStatus.FORBIDDEN);
 		}
 		
 		// Validate tokens received from BCSC. 
 		logger.debug("Validating ID token received from BCSC...");
 		ValidationResponse valResp = tokenServices.validateBCSCIDToken((String)token.toSuccessResponse().getCustomParameters().get("id_token"));
 		if (!valResp.isValid()) {
-			logger.error("ID token failed to validate. Error: " + valResp.getMessage());
+			logger.error("ID token failed to validate. Error {}", valResp.getMessage());
```

Comment:
Just curious, why the { } ?

Reply:
That is how the logger will append the string into the output. It is similar to using %s in string.format.

Reply:
OK, cool. didn't know this. Thanks.